tests/Benchmarks/System.Text.Primitives/AsciiCasing.cs

```diff
@@ -0,0 +1,34 @@
+﻿using BenchmarkDotNet.Attributes;
+using System.Buffers;
+
+namespace System.Text.Primitives.Benchmarks
+{
+    public class AsciiCasing
+    {
+        private byte[] bytes;
```

Comment:
nit: private fields need to start with underscore. Please rename to _bytes, and _output.
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

> 3. We use `_camelCase` for internal and private fields and use `readonly` where possible. Prefix internal and private instance fields with `_`, static fields with `s_` and thread static fields with `t_`. When used on static fields, `readonly` should come after `static` (e.g. `static readonly` not `readonly static`).  Public fields should be used sparingly and should use PascalCasing with no prefix when used.